### DISCUSSION TITLE
Do slingbox_server.exe or slingbox_server.py take any parameters?

### DISCUSSION THREAD

**Discussion by DVoorhees1957** [GerryDazoo/Slinger#231] [Q&A] [answered]
I know that slingbox_server.py needs at least 1 parameter, the name of a config file. Does slingbox_server.exe need that same thing? Can either of them take a parameter that specifies where to log any output?

I've looked through the readme, the 3.08 release notes, and the networking guide, and I didn't see anything documented.

**Comment by GerryDazoo** [comment, marked as answer]
Technically it doesn't need any parameters, it will default to "config.ini" if nothing is supplied.
You can pass in mutiple config files. config1.ini config2.ini ..... But in this case I'd recommend using the unified_config.ini file format supported in the latest release.
There is no log file parameter. Just use your OS supplied capabilities to redirect the output to a file.
i.e.  slingbox_server.exe  > /path/to/somefile

**Reply by DVoorhees1957** [reply]
Thanks!
